decidim-dev/lib/decidim/dev/test/rspec_support/feature.rb

```diff
@@ -7,8 +7,15 @@ class DummyEngine < Rails::Engine
 
     routes do
       root to: proc { [200, {}, ["DUMMY ENGINE"]] }
+      resources :dummy_resource
     end
   end
+
+  class DummyResource < ActiveRecord::Base
```

Comment:
Maybe, but after having to deal with it and understanding it felt that it was easier to use it and modify it when everything is together. Otherwise it might be difficult to see the whole picture. I can split it though if you think it's worth it.